checkstyle-checks/src/main/resources/sonatype/checkstyle-configuration.xml

```diff
@@ -0,0 +1,181 @@
+<!--
+
+    Copyright (c) 2011-present Sonatype, Inc. All rights reserved.
+    Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+    "Sonatype" is a trademark of Sonatype, Inc.
+
+-->
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+<module name="Checker">
+  <property name="charset" value="UTF-8"/>
+
+  <property name="severity" value="error"/>
+
+  <property name="fileExtensions" value="java"/>
+  <module name="FileTabCharacter">
+    <property name="eachLine" value="true"/>
+  </module>
+
+  <module name="NewlineAtEndOfFile"/>
+
+  <module name="SuppressWarningsFilter" />
+
+  <module name="TreeWalker">
+    <module name="com.sonatype.checks.ClassStructureEmptyLineCheck" />
+    <module name="SuppressWarningsHolder" />
+    <module name="UnnecessaryParentheses"/>
```

Comment:
I think I've said it before, but I'm too lazy to go back and search for my comments, so here I go again:
In my opinion, we should enforce only the rules that are meaningful for us. The rules that will catch real problems and not restrict us too much. The rules that will catch problems that we currently have to find the hard way.

For this particular rule... I simply don't remember when I've seen some unnecessary parentheses that needed to be removed. Which makes me ask why we want to enforce a rule that doesn't fix real problems in our code.

Reply:
> For this particular rule... I simply don't remember when I've seen some unnecessary parentheses that needed to be removed.

https://github.com/sonatype/insight-brain/blob/5308125edfd0337f81e08065fccc22c0724f3930/insight-brain-service/src/main/java/com/sonatype/insight/brain/product/license/CLMLicenseManager.java#L386